tag-release-candidate.sh

```diff
@@ -23,10 +23,10 @@ if ! [[ "${release_branch}" =~ ^release-[0-9]+\.[0-9]+$ ]] ; then
   exit 3
 fi
 
-if git remote get-url origin | grep -v "git@github.com:weaveworks/eksctl" ; then
+if [ ! "$(git remote get-url origin)" = "git@github.com:weaveworks/eksctl" ] ; then
```

Comment:
FYI, this change broke the release scripts on my machine.

**Given**:
```console
$ git remote get-url origin
git@github.com:weaveworks/eksctl.git
```

**Before**:
```console
$ git remote get-url origin | grep -v "git@github.com:weaveworks/eksctl"
$ echo $?
1
```
hence no error, and the script was running fine.
**After**:
```console
$ [ ! "$(git remote get-url origin)" = "git@github.com:weaveworks/eksctl" ] && echo "KO"
KO
```
hence:
```console
$ ./tag-release-candidate.sh 0.9.0-rc.0
Invalid origin: git@github.com:weaveworks/eksctl.git
```